examples/aes_main.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * #%L
+ * %%
+ * Copyright (C) 2020 BMW Car IT GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+#include "common.h"
+#include "aes_example.h"
+#include "mococrw/private/IOUtils.h"
+#include "mococrw/private/parsingUtils.h"
+
+namespace po = boost::program_options;
+using namespace mococrw;
+
+void parseCommandlineArgs(int argc, char *argv[], po::variables_map &vm)
+{
+    po::options_description genericAesOptions("Options for symmetric en-/decryption");
+    genericAesOptions.add_options()
+        ("key", po::value<std::string>(), "The key which shall be used for encryption/decryption in hex form (with or "
+                                          "without 0x prepended). Two sizes are supported: 128 and 256 bits (16 and 32 "
+                                          "bytes).")
+        ("operation-mode", po::value<std::string>(), "The operation mode. Supported: GCM, CTR, CBC. GCM is the only "
+                                                     "mode supporting Authenticated Encryption (AE)")
+        ("padding", po::value<std::string>()->default_value("PKCS"), "The padding mode. Optional. "
+                                                                     "Default: PKCS Supported: PKCS, NO")
+        ("iv", po::value<std::string>(), "The IV value used for en-/decryption in hex form (with or without 0x "
+                                         "prepended). If empty a random value is used for encryption. Default: empty. "
```

Comment:
```suggestion
                                         "prepended). If empty, a random value is used for encryption. Default: empty. "
```
I would add a comma here, this makes it easier to get what you meant on the first "parse attempt"